src/instructlab/model/dk_bench_utils.py

```diff
@@ -0,0 +1,310 @@
+    gpus: int | None,
+    backend: str | None,
+    enable_serving_output: bool,
+) -> tuple:
+    server = None
+    if model_is_localand_valid(model):
+        logger.debug("DK-Bench model is local")
+        model_name = get_model_name(model)
+        server, api_base, _ = launch_server(
+            eval_serve=ctx.obj.config.serve,
+            tls_client_cert=ctx.params["tls_client_cert"],
+            tls_client_key=ctx.params["tls_client_key"],
+            tls_client_passwd=ctx.params["tls_client_passwd"],
+            tls_insecure=ctx.params["tls_insecure"],
+            model=model,
+            model_name=model_name,
+            max_workers=max_workers,
+            gpus=gpus,
+            backend=backend,
+            enable_serving_output=enable_serving_output,
+        )
+    else:
+        logger.debug("DK-Bench model is an endpoint")
+        http_params = http_client(
+            {
+                "tls_client_cert": ctx.params["tls_client_cert"],
+                "tls_client_key": ctx.params["tls_client_key"],
+                "tls_client_passwd": ctx.params["tls_client_passwd"],
+                "tls_insecure": ctx.params["tls_insecure"],
+            }
+        )
+
+        api_key = os.environ.get("STUDENT_ENDPOINT_API_KEY", None)
+        if api_key is None:
+            logger.debug(
+                "API_KEY for model at endpoint %s is not set. To set it set the environment variable $STUDENT_ENDPOINT_API_KEY",
+                model,
+            )
+
+        model_name = get_endpoint_model_name(model, api_key, http_params)
+        api_base = model
+
+    openai_client = get_openai_client(model_api_base=api_base, api_key=api_key)
+    model_config = ModelConfig(
+        model_name=model_name, temperature=temperature, system_prompt=model_prompt
+    )
+
+    return server, model_config, openai_client
+
+
+def make_run_dir(output_dir: str) -> str:
+    now = datetime.now()
+    timestamp = now.strftime("%m-%d-%Y_%H-%M-%S-%f")
+    if output_dir.endswith("/"):
+        output_dir = output_dir[:-1]
+    run_dir = f"{output_dir}/job-{timestamp}"
+
+    path_does_not_exist = not os.path.exists(output_dir)
+    path_is_not_dir = not os.path.isdir(output_dir)
+    if path_does_not_exist or path_is_not_dir:
+        run_dir = f"{DEFAULTS.EVAL_DATA_DIR}/job-{timestamp}"
+
+    os.makedirs(run_dir, exist_ok=True)
+    return run_dir
+
+
+def print_header():
+    print("\n")
+    print("# DK-BENCH REPORT")
+
+
+def print_results(result: EvaluationResult, model_name: str):
+    print(f"\n## MODEL: {model_name}\n")
+    total_score = 0
+    question_num = 0
+    for score in result.scores:
+        question_num += 1
+        print(f"Question #{question_num}:     {score['domain_specific_rubrics']}/5")
+        total_score += score["domain_specific_rubrics"]
+
+    average = total_score / len(result.scores)
+    average = round(average, 2)
+    print("----------------------------")
+    print(f"Average Score:   {average}/5")
```

Comment:
Instead of averaging you can simply use formatting syntax
```suggestion
    print("----------------------------")
    print(f"Average Score:   {average:.2f}/5")
```